[2023-12-12T09:20:53.803+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-12-12T09:20:53.824+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-23T00:00:00+00:00 [queued]>
[2023-12-12T09:20:53.825+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-12T09:20:53.843+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): call_api_and_process> on 2023-01-23 00:00:00+00:00
[2023-12-12T09:20:53.849+0000] {standard_task_runner.py:57} INFO - Started process 3694 to run task
[2023-12-12T09:20:53.852+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_bipo_dag', 'call_api_and_process', 'scheduled__2023-01-23T00:00:00+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/bipo_employee_info.py', '--cfg-path', '/tmp/tmplz775wml']
[2023-12-12T09:20:53.855+0000] {standard_task_runner.py:85} INFO - Job 216: Subtask call_api_and_process
[2023-12-12T09:20:53.876+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-12-12T09:20:53.919+0000] {task_command.py:416} INFO - Running <TaskInstance: my_bipo_dag.call_api_and_process scheduled__2023-01-23T00:00:00+00:00 [running]> on host e1a26a98da8f
[2023-12-12T09:20:54.045+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_bipo_dag' AIRFLOW_CTX_TASK_ID='call_api_and_process' AIRFLOW_CTX_EXECUTION_DATE='2023-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-23T00:00:00+00:00'
[2023-12-12T09:23:55.756+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-12-12T09:23:55.759+0000] {process_utils.py:131} INFO - Sending 15 to group 3694. PIDs of all processes in the group: [3694]
[2023-12-12T09:23:55.759+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 3694
[2023-12-12T09:23:55.760+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-12-12T09:23:55.812+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=3694, status='terminated', exitcode=0, started='09:20:53') (3694) terminated with exit code 0
